Rem/Commands/PatModule.cs

```diff
@@ -34,19 +33,19 @@ public PatModule(SQLiteAsyncConnection dbConnection, BotState state)
         }
 
         [Command("pat"), Summary("Headpats a user (or yourself if not specified)")]
-        public async Task Pat([Summary("The person to headpat")] IUser user = null, [Remainder] string text = null)
```

Comment:
Even though text is unused, the behavior is different if it is not present because of the [Remainder] attribute. It should not be removed.